src/format-dependencies.js

```diff
@@ -2,7 +2,8 @@
 
 import type { PackageWithDeps, PackageAtVersion, PackageVersionMap } from './types';
 
-function resolveDependencies({ dependencies, resolvedDependencies }: {
+function resolveDependencies({
+  dependencies, resolvedDependencies }: {
```

Comment:
Is this from prettier? I'm confused as to why you'd break an object onto two lines? Feel like the original was cleaner.

Reply:
That's `eslint --fix`

You could add prettier if you wanted to though

Reply:
ewww